pyintegration/tests/test_constants.py

```diff
@@ -0,0 +1,18 @@
+#
+#   Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#
+
+from deephaven2.constants import *
+from tests.testbase import BaseTestCase
+
+
+class ConstantsTestCase(BaseTestCase):
+
+    def test_null_values(self):
```

Comment:
These tests almost do nothing but check if JPY maps these Java values correctly to Python values. So they are not necessary if we trust JPY. However it might still be comforting to know the translated values of these Java constants.